jobs/RunCwr/config.xml

```diff
@@ -58,7 +58,7 @@ function cleanup_model() {
 }
 trap cleanup_model EXIT
 
-cwr -F $CONTROLLER:test-cwr-$CHARM_NAME-$BUILD_NUMBER totest.yaml --results-dir /srv/artifacts --test-id $BUILD_NUMBER
+env MATRIX_OUTPUT_DIR=/srv/artifacts cwr -F $CONTROLLER:test-cwr-$CHARM_NAME-$BUILD_NUMBER totest.yaml --results-dir /srv/artifacts --test-id $BUILD_NUMBER
```

Comment:
Why is `MATRIX_OUTPUT_DIR` needed here?  I think it would be nicer if cwr set the MATRIX_OUTPUT_DIR whenever it spawned matrix.  At that point, cwr would know the path (from --results-dir and --test-id params) and could just drop the results into the appropriate job dir without any extra env from this charm layer.